CMakeLists.txt

```diff
@@ -35,11 +35,11 @@ cuda_add_executable(
     ${moderngpu_source}
 )
 
-cuda_add_executable(
-    demo-sort
-    utils/modern-sort.cu
-    ${moderngpu_source}
-)
+#cuda_add_executable(
```

Comment:
Just because I thought I don't use demo-sort, and hence it is not needed for me.
This PR was not intended to be merged directly. I'm sorry for confusing if so..